app/tests.py

```diff
@@ -136,8 +138,7 @@ def setUp(self):
             'organizer_email': 'pepe@planner.com',
             'signed_date': '2019-09-14',
         }
-        contract = Contract(**contract_data)
-        contract.save()
+        contract = Contract.objects.create(**contract_data)
```

Comment:
Have you consider using `factory_boy`? It will simplify all your tests, you can create django models objects easily. Usage:
Creating factories: https://github.com/evbeda/imahunga/blob/master/bundesliga_app/factories.py
Using factories in tests: https://github.com/evbeda/imahunga/blob/master/bundesliga_app/tests.py#L256-L271